### ISSUE TITLE
[DOCS] AUR packages

### ISSUE THREAD

**Issue by otaj** [kornia/kornia#289]
Hi,

it's not really an issue, I just wanted to let you know I made AUR (Arch User Repository) packages for kornia, available [here](https://aur.archlinux.org/packages/python-kornia/) for normal releases and [here](https://aur.archlinux.org/packages/python-kornia-git/) for git version.

Keep up the good work :-)

**Comment by edgarriba** [comment]
@otaj this is great :D Planning very soon to add a link with projects or available resource to get the lib. Leave this open by now. Updating news with it too.

**Comment by edgarriba** [comment]
Notice that we have moved the project to kornia org :D

https://github.com/kornia

**Comment by otaj** [comment]
Packages have been updated with new URL. I left a github there (github.com/kornia/kornia), since then it is easier to keep the URL and source tied together ;) If you wish, I can change it to kornia.org

**Comment by edgarriba** [comment]
github is fine :D

**Comment by edgarriba** [comment]
@otaj do you have any automatic tool to update packages each time we do a realease ? Notice that v0.2.0 is out !

**Comment by otaj** [comment]
Hi, I just updated the package on AUR. I do not any automatic tool to do so, however I was planning to set something like that, so I will get to it right away :) Thanks for such a reminder.

BTW, variable `version` in `setup.py` in release archive is 0.1.2, I believe it should read 0.2.0. It just means, that the `import kornia; print(kornia.__version__)` gives something like 0.1.2

**Comment by edgarriba** [comment]
I have to rethink this part about versioning. However, each time I do a release I put by hand the version number. So the wheel from pypi should be fine.

**Comment by otaj** [comment]
Well, the `setup.py` in tag v0.2.0 in repo has a wrong version, so maybe it just was pushed after it was properly tagged?

**Comment by edgarriba** [comment]
Well is already fixed :)

**Comment by edgarriba** [comment]
@otaj Is there any way we can show the usage of this ? We have a blog now where would be great to have a small tutorial to see how is this used

**Comment by otaj** [comment]
@edgarriba Usage of what? AUR package? You mean installation instructions on Arch? `<your favourite AUR helper> -S python-kornia` or `<your favourite AUR helper> -S python-kornia-git` for git version.

Since `yay` is currently probably most often used AUR helper, you can even simplify it to `yay -S python-kornia` or `yay -S python-kornia-git`

Is this what you had in mind?

**Comment by edgarriba** [comment]
yeah probably, a short entry explaining what exactly AUR or Arch are, showing links (or screenshots) wth few commands.
Maybe some users don't even know how to use or the existence of AUR.

Check our blog - https://github.com/kornia/kornia-blog

You can easily create a post/tutorial with simple markdown sending a PR to our github
https://github.com/kornia/kornia-blog/blob/master/_posts/2020-08-07-kornia-release-v040.md

**Comment by otaj** [comment]
AUR (Arch User Repository) is a system of sharing packages for Arch Linux, one of Linux distributions (i.e. same relationship as PPA (personal package archive) for Ubuntu).

I doubt that people, who are using Arch, don't know, what AUR is and people, who are not using Arch will not use AUR anyway.

I will definitely not create blog post explaining what is Arch or AUR for kornia, that just doesn't make sense, there are infinitely better places for such a blog post. It would be the same as having a blog post on CV library blog explaining what is Ubuntu or Windows.

**Comment by edgarriba** [comment]
@otaj I'll leave this issue open in case anyone want to create a tutorial on that.

In any case, thanks for maintaining the package.
